BraveWallet/WalletStrings.swift

```diff
@@ -1513,5 +1513,26 @@ extension Strings {
       value: "Quote includes a %@ Brave fee.",
       comment: "A disclaimer that appears at the bottom of an swap screen which discloses the fixed Brave fee included in the swap quotes. '%@' will be replaced by a percentage. For example: 'Quote includes a 0.875% Brave fee'"
     )
+    public static let screenshotDetectedTitle = NSLocalizedString(
+      "wallet.screenshotDetectedTitle",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Screenshot detected",
+      comment: "A title of an alert when the user takes a screenshot of their device"
+    )
+    public static let recoveryPhraseScreenshotDetectedMessage = NSLocalizedString(
+      "wallet.recoveryPhraseScreenshotDetectedMessage",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "If you use a cloud service to backup files / photos, a screenshot of your recovery phrase may have just been uploaded. This could expose you to risk.",
+      comment: "The message displayed when the user takes a screenshot of their recovery phrase"
+    )
+    public static let privateKeyScreenshotDetectedMessage = NSLocalizedString(
+      "wallet.privateKeyScreenshotDetectedMessage",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "If you use a cloud service to backup files / photos, a screenshot of your private key may have just been uploaded. This could expose you to risk.",
```

Comment:
Cc: @orspetol

Reply:
Cc: @rmcfadden3

Reply:
I believe something like that works better: "WARNING: Screenshots of the private key may end in cloud backups and be readable by any application with Photos access. Please don't save the screenshot and delete it as soon as possible!"

Reply:
Same as above